packages/calling/src/CallingClient/registration/register.ts

```diff
@@ -151,6 +150,7 @@ export class Registration implements IRegistration {
       },
     });
 
+    this.setStatus(RegistrationStatus.INACTIVE);
```

Comment:
Wouldn't it be better to move this to IDLE? We are deleting the registration on purpose here, so ideally we (developer) already know that the registration is inactive.
I feel INACTIVE is generally a state to be used when the registration has been deleted from the mobius side or we receive an error in registration.

Reply:
As per our discussion, IDLE status is removed for now. We only have Active and Inactive.

Reply:
Clarified now.